### PR TITLE
Create attributes-on-lambda-parameters

### PR DESCRIPTION
**Pull request by JeWaVe**

### CODE REVIEW

**Comment by HaloFour** [comment]
You should open a new proposal in the Issues section.  Any language proposal needs to have a "champion" from the language design team before it will be considered for addition to this repository.

**Comment by alrz** [comment]
duplicate of https://github.com/dotnet/csharplang/pull/375

**Comment by jcouv** [comment]
@MadsTorgersen This is a proposal for a language feature. Let's triage when possible in LDM and see if someone is ready to champion.

@JeWaVe Thanks for writing this up.

File: proposals/attributes-on-lambda-parameters [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Attributes on anonymous lambda parameters
```
Comment (jamesqo) [inline proposals/attributes-on-lambda-parameters]: You forgot to suffix the file with `.md`.

**Comment by RikkiGibson** [comment]
Apologies, this PR was closed automatically as part of renaming master->main.

**Pull request closed without merging**